core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages.component.scss

```diff
@@ -13,3 +13,19 @@
     padding: $spacing-4 $spacing-5 0;
     width: 100%;
 }
+
+.dot-pages__spinner-container {
+    height: 100%;
+    position: relative;
+}
+
+.dot-pages__spinner-overlay {
+    align-content: center;
+    background-color: $white;
+    display: flex;
+    flex-wrap: wrap;
+    height: calc(100% - $spacing-7);
+    justify-content: center;
+    position: absolute;
+    width: calc(100% - $spacing-8);
+}
```

Comment:
Please add a video on this loading indicator and check in with Melissa if it should look like this globally?

Reply:
video uploaded
https://user-images.githubusercontent.com/37185433/231851175-4cf40668-db36-44a0-96da-af3b1b56945b.mov

Reply:
@fmontes after discussing with Melissa, the style was updated!

https://user-images.githubusercontent.com/37185433/231879476-5628cfb6-df5b-4ec2-8988-6ea7022bdfae.mov